components/DesaDigital/Detail/index.vue

```diff
@@ -0,0 +1,245 @@
+        </template>
+      </BaseButton>
+    </div>
+    <div class="desa-digital-detail__content">
+      <h1 class="desa-digital-detail__content-header">
+        Identitas Pemohon
+      </h1>
+      <jds-simple-table class="desa-digital-detail__content-table">
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Nama Lengkap</strong>
+              <span>:</span>
+            </td>
+            <td>{{ item.properties.applicant.name || '-' }}</td>
+          </tr>
+          <tr>
+            <td>
+              <strong>No HP</strong>
+              <span>:</span>
+            </td>
+            <td>{{ item.properties.applicant.phone_number || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Email</strong></td>
+            <td>{{ item.properties.applicant.email || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+    </div>
+    <div class="desa-digital-detail__content">
+      <h1 class="desa-digital-detail__content-header">
+        Detail Desa {{ item.village.name || '' }}
+      </h1>
+      <jds-simple-table class="desa-digital-detail__content-table">
+        <thead>
+          <tr>
+            <th colspan="2">
+              Tentang Desa
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Nama</strong>
+            </td>
+            <td>{{ item.village.name || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Kecamatan</strong></td>
+            <td>{{ item.district.name || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Kabupaten/Kota</strong></td>
+            <td>{{ item.city.name || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+      <jds-simple-table class="desa-digital-detail__content-table">
+        <thead>
+          <tr>
+            <th colspan="2">
+              Fasilitas Desa
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Akses Kendaraan</strong>
+            </td>
+            <td>{{ vehicles }}</td>
+          </tr>
+          <tr>
+            <td><strong>Suplai Listrik</strong></td>
+            <td>{{ item.properties.facility.power_supply.data || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Jaringan Telepon</strong></td>
+            <td>{{ item.properties.facility.cellular_network.data || '-' }}</td>
+          </tr>
+          <tr>
+            <td><strong>Jaringan Intener</strong></td>
+            <td>{{ item.properties.facility.internet_network.data || '-' }}</td>
+          </tr>
+        </tbody>
+      </jds-simple-table>
+      <jds-simple-table v-show="item.level === 2 || item.level === 3 || item.level === 4" class="desa-digital-detail__content-table">
```

Comment:
discussion:
maybe we can make a computed function or declare a variable in data that more readable or easy to understand such as
`showLiteration: item.level === 2 || item.level === 3 || item.level === 4`

also applies to others such as showPotency, etc., what do u think?

Reply:
I agree we can create a computed function for each condition. We can give the name like `showLiteration, showBumdes, and showPotency`, wdyt?

Reply:
sure, another option is classify / alias it by it's level first (isLevelOne, two, ...), and then showLiteration decision is based on it's  questionnaire level.
Whatever that is, I will leave it up to your decision 😄

Reply:
The change is [here](https://github.com/jabardigitalservice/dedi-cms/pull/92/commits/541ddef9bca435b3d0caf540c0f2d0585a3b0e88) ⭐